docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -91,6 +93,18 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
     Filters the input rows for which the `boolean_expression` in the `WHERE` clause evaluates
     to true are passed to the aggregate function; other rows are discarded.
 
+* **FIRST**
+
+    `FIRST` selects a first expression value from the data set. We can specify an optional `IGNORE NULL` clause to ignore NULL values.
```

Comment:
I thought data set is one word. I searched docs, both data set and dataset are used, so I guess either way is OK.